### PR TITLE
docs: removed link from opera icon in README

### LINKED ISSUE #416: [docs] remove opera logo's link in README
### Description

Opera can install chrome extension in specific way actually, but an opera logo with chrome webstore link is confusing:

image

Better to remove the link from the logo (in two README files):

image

### PR DESCRIPTION
**Pull request by debjit20504**
## Brief Information

This pull request is in the type of:

- [ ] bug fixing
- [ ] new feature
- [x] docs

Related issues:

- resolve #416 

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/hypertrons/hypertrons-crx)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,27 +17,27 @@ You can find these dashboards in:
...
+	  </td>
+	  <td>
+		<img
+		  src="https://user-images.githubusercontent.com/32434520/180447103-76ff1e25-ec35-4e7f-bd54-9d98545ca1df.png"
+		/>
+		<img
+		  src="https://user-images.githubusercontent.com/32434520/180446790-50b6a53b-119e-4b74-a08d-dda146fb9f29.png"
+		/>
+	  </td>
+	</tr>
```
Comment (tyn1998) [inline README.md:40]: Hi, @debjit20504, it seems that the markdown editor you use automatically formatted the markdown code. I hope you can keep the unrelated code as they were before.

Could you make changes and push another commit to this PR?

**Review by tyn1998 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (debjit20504) [reply README.md:40]: ok, I'll do that

Comment (debjit20504) [reply README.md:40]: [README.md](https://github.com/hypertrons/hypertrons-crx/files/9173238/README.md)
[README.zh-CN.md](https://github.com/hypertrons/hypertrons-crx/files/9173239/README.zh-CN.md)

**Review by debjit20504 — COMMENTED** [review]
removed link from opera icon and the rest code is as they were before in README.md and README.zh-CN.md

**Review by tyn1998 — APPROVED** [review]
Hi, @debjit20504,

It seems that you formatted all html tags with 2 indents to "revert" changes that your editor made. Though it is not what I meant I really like the formatted code now.

LGTM. Thank you!

/approve

**Comment by debjit20504** [comment]
Thanks :) 👍

**Comment by tyn1998** [comment]
In case you are not familiar with the whole process to contribute to this project, you are encouraged to read our [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md). 

If you had created another branch for this PR (which is more practical), you would have done the requested changes more easily -- just delete the commits you made to this PR branch and remake changes then add, commit and force push the commit to this PR, done!
